README.md

```diff
@@ -48,30 +48,38 @@ the [releases page](https://github.com/Weave-MC/Weave-Loader/releases), or build
 
 ### Building with Gradle
 
-- `git clone` the project, this can be achieved by installing [git][git], then running
+- First clone the project with [Git][git] then `cd` into the project directory: 
 
 ```bash
-git clone https://github.com/Weave-MC/Weave-Loader.git
-git submodule update --init --recursive
+git clone --recursive https://github.com/Weave-MC/Weave-Loader
+cd Weave-Loader
 ```
 
-- **UN*X**
+You then need to give permission to the Gradle wrapper and run the `agent` task. This can be done a bit differently
+depending on your operating system:
+
+<details open>
+<summary><b>UN*X</b> (Linux, BSDs, macOS, etc.)</summary>
 
 ```bash
-cd Weave-Loader && chmod +x ./gradlew && ./gradlew agent
+chmod +x ./gradlew
```

Comment:
Git does track permissions, I just kept it since it was already in, but also because committing on Windows sometimes unset those permission flags, so this is just a safe measure. Should I remove it?